packages/core-app-api/src/lib/AuthConnector/DefaultAuthConnector.ts

```diff
@@ -82,9 +87,18 @@ export class DefaultAuthConnector<AuthSession>
 
     this.authRequester = oauthRequestApi.createAuthRequester({
       provider,
-      onAuthRequest: scopes => this.showPopup(scopes),
+      onAuthRequest: async scopes => {
+        const enableExperimentalRedirectFlow =
+          this.configApi.getOptionalBoolean('enableExperimentalRedirectFlow') ??
+          false;
```

Comment:
I think it's a bit nicer to read this once in the constructor instead, and then store it in a member